packages/tailwind/src/_docs/pages/configuration.mdx

```diff
@@ -1,5 +1,5 @@
 import { Meta } from "@storybook/blocks"
-import { InlineNotification } from "~components/Notification"
+import { InlineNotification } from "@kaizen/components"
```

Comment:
Given it is only docs, and this creates a possible circular dependency, it might be safer to have this as a local alias

Reply:
Works for MDX but fails for the tsx stories, I think it works fine though